scripts/lib/libvirt/compute-config

```diff
@@ -32,6 +32,8 @@ def main():
                         help="Virtual Disk Directory (e.g /dev/cloud)")
     parser.add_argument("bootorder", type=str,
                         help="Boot Order (e.g. 2)")
+    parser.add_argument("numcontrollers", type=str,
+                        help="Number of controller nodes")
```

Comment:
Please no. You are using this value to compare "1" with "1", as string when the semantic is integer. Is not a matter of style: is a bug

Maybe we need to fix the rest of the parameters in a different PR, but do not add a new bug here.